experimental/framework/frs-client/src/FrsClient.ts

```diff
@@ -114,53 +115,61 @@ export class FrsClientInstance {
 }
 
 /**
- * FrsClient static class with singular global instance that lets the developer define
- * all Container interactions with the Frs service
+ * FrsClient provides the ability to have a Fluid object backed by the FRS service or, when running with
+ * localMode enabled, have it be backed by Tinylicious locally
  */
-// eslint-disable-next-line @typescript-eslint/no-extraneous-class
 export class FrsClient {
-    private static globalInstance: FrsClientInstance | undefined;
+    private readonly clientInstance: FrsClientCore | TinyliciousClient;
 
-    static init(connectionConfig: FrsConnectionConfig) {
-        if (FrsClient.globalInstance) {
-            debug(
-                `FrsClient has already been initialized. Using existing instance of
-                FrsClient instead.`,
-            );
-            return;
+    constructor(private readonly connectionConfig: FrsConnectionConfig) {
+        if (connectionConfig.type === "localMode") {
+            this.clientInstance = new TinyliciousClient();
+        }
+        else {
```

Comment:
nit: not sure the linter catches this but this should be on the same line for consistency